web/index.html

```diff
@@ -312,14 +317,15 @@ <h2 class="f4">What does it mean if I get an error?</h2>
             }
         })
 
-        outText += `${successfulProviders > 0 ? '✅' : '❌'} Found ${successfulProviders} working providers (out of ${resp.length} provider records sampled from Amino DHT) that could be connected to and had the CID available over Bitswap:`
+        outText += `${successfulProviders > 0 ? '✅' : '❌'} Found ${successfulProviders} working providers (out of ${resp.length} provider records sampled from Amino DHT and IPNI) that could be connected to and had the CID available over Bitswap:`
         for (const provider of resp) {
             const couldConnect = provider.ConnectionError === ''
 
             outText += `\n\t${provider.ID}\n\t\tConnected: ${couldConnect ? "✅" : `❌ ${provider.ConnectionError.replaceAll('\n', '\n\t\t')}` }`
             outText += couldConnect ? `\n\t\tBitswap Check: ${provider.DataAvailableOverBitswap.Found ? `✅` : "❌"} ${provider.DataAvailableOverBitswap.Error || ''}` : ''
             outText += (couldConnect && provider.ConnectionMaddrs) ? `\n\t\tSuccessful Connection Multiaddr${provider.ConnectionMaddrs.length > 1 ? 's' : ''}:\n\t\t\t${provider.ConnectionMaddrs?.join('\n\t\t\t') || ''}` : ''
             outText += (provider.Addrs.length > 0) ? `\n\t\tPeer Multiaddrs:\n\t\t\t${provider.Addrs.join('\n\t\t\t')}` : ''
+            outText += `\n\t\tFound in: ${provider.Source}`

```

Comment:
Seems fine to add, but do we care?

Reply:
Meh 🤷 
Might be interesting to someone.

Reply:
> It looks like right now for IPNI we're not answering the 4th question

This is handled now.

Reply:
This is very useful info, as user I can see that only reason things work is because of IPNI, and I lack Amino providers (or vice versa). Especially if I paid someone to announce to specific routing systems, or configured announcements myself (but incorrectly, or falling behind).